test-integ/catalogv2/explicit_destinations_test.go

```diff
@@ -152,7 +152,9 @@ func (c testBasicL4ExplicitDestinationsCreator) topologyConfigAddNodes(
 				clusterName,
 				newID("single-server"),
 				topology.NodeVersionV2,
-				nil,
```

Comment:
Changed the 4 services in this test to use an overridden workload identity field to showcase that it works now.